src/Microsoft.Health.Dicom.Core/Features/CustomTag/CustomTagStoreEntry.cs

```diff
@@ -10,11 +10,12 @@ namespace Microsoft.Health.Dicom.Core.Features.CustomTag
     /// </summary>
     public class CustomTagStoreEntry
     {
-        public CustomTagStoreEntry(int key, string path, string vr, CustomTagLevel level, CustomTagStatus status)
+        public CustomTagStoreEntry(int key, string path, string vr, string privateCreator, CustomTagLevel level, CustomTagStatus status)
```

Comment:
All of the `CustomTagStoreEntries` aren't private are they? So should this optional?

Reply:
I think it's fine to put it in the end. my perspective is CustomTagStoreEntry represent an entry in database, so just mapping to whatever in database.